src/pybind/mgr/requirements-required.txt

```diff
@@ -14,3 +14,4 @@ requests-mock
 scipy
 werkzeug
 natsort
+packaging
```

Comment:
please note the rationales in the commit message, and would be better to note the offending commit in the commit message also.